setup/helpers/perf_helper.rb

```diff
@@ -77,6 +77,7 @@ def perf_install_pe
       r10k_remote = '/opt/puppetlabs/server/data/puppetserver/r10k/control-repo'
       @options[:answers] ||= {}
       @options[:answers]['puppet_enterprise::profile::master::r10k_remote'] = r10k_remote
+      @options[:answers]['puppet_enterprise::profile::puppetdb::node_ttl'] = '0s'
```

Comment:
Is there a reason we aren't just putting this in the control repo?

Reply:
The control repo does not manage the puppet_enterprise module. That module is managed by the installer. It lives in /opt/puppetlabs/puppet/modules which is out of scope for the control repo.

Reply:
@RandellP I have tried adding this to the `hieradata/common.yml` file and I do not see any evidence that the setting is picked up by the `puppet_enterprise` module. I would expect to see the puppetdb config files updated with this value or the parameter to the 'PE PuppetDB' node group in the console. Neither one of these is the case.
```
[root@ip-10-227-3-114 hieradata]# grep -r node_ttl /etc/puppetlabs/
/etc/puppetlabs/code/environments/production/hieradata/common.yaml:puppet_enterprise::profile::puppetdb::node_ttl: '0s'
/etc/puppetlabs/code-staging/environments/production/hieradata/common.yaml:puppet_enterprise::profile::puppetdb::node_ttl: '0s'
[root@ip-10-227-3-114 hieradata]# puppet agent -t
Info: Using configured environment 'production'
Info: Retrieving pluginfacts
Info: Retrieving plugin
Info: Retrieving locales
Info: Loading facts
Error: Facter: error while resolving custom fact "i18ndemo_fact": i18ndemo_fact: this is a raise from a custom fact from eputnam-i18ndemo
Info: Caching catalog for ip-10-227-3-114.amz-dev.puppet.net
Info: Applying configuration version '1556143833'
Notice: Applied catalog in 12.86 seconds
[root@ip-10-227-3-114 hieradata]# grep -r node_ttl /etc/puppetlabs/
/etc/puppetlabs/code/environments/production/hieradata/common.yaml:puppet_enterprise::profile::puppetdb::node_ttl: '0s'
/etc/puppetlabs/code-staging/environments/production/hieradata/common.yaml:puppet_enterprise::profile::puppetdb::node_ttl: '0s'
```